tests/test_details_component_has_js.py

```diff
@@ -0,0 +1,11 @@
+from glob import glob
```

Comment:
Could we move these tests under `tests/src`? This would be consistent with the other tests in that directory, and avoid cluttering the top level.

Reply:
Fixed in https://github.com/alphagov/notifications-govuk-alerts/pull/163/commits/d9103f678c4511538055eb91c8832b43c23f5e37 and https://github.com/alphagov/notifications-govuk-alerts/pull/163/commits/fa45e96a122aab3212b6e34ab1053b5c71f39c26